src/addressBook/index.test.js

```diff
@@ -271,47 +272,73 @@ describe("getContacts", () => {
     jest.restoreAllMocks();
   });
   test("it fetches the people in the address book", async () => {
-    const containerIri = "https://user.example.com/contacts";
+    const mockIndexFileDatasetIri =
+      "https://user.example.com/contacts/people.ttl";
+    const mockIndexFileDataset = solidClientFns.mockSolidDatasetFrom(
+      mockIndexFileDatasetIri
+    );
     const fetch = jest.fn();
     const personContainer1 = "https://user.example.com/contacts/Person/1234/";
     const personContainer2 = "https://user.example.com/contacts/Person/5678/";
     const expectedPerson1 = {
-      dataset: "Person 1",
+      dataset: mockPersonContactDataset(),
       iri: `${personContainer1}index.ttl`,
     };
     const expectedPerson2 = {
-      dataset: "Person 2",
+      dataset: mockPersonContactDataset(),
       iri: `${personContainer2}index.ttl`,
     };
 
     jest
       .spyOn(resourceFns, "getResource")
-      .mockResolvedValueOnce({ response: { dataset: "people container" } })
       .mockResolvedValueOnce({ response: expectedPerson1 })
       .mockResolvedValueOnce({ response: expectedPerson2 });
 
     jest
-      .spyOn(solidClientFns, "getUrlAll")
-      .mockReturnValueOnce([expectedPerson1.iri, expectedPerson2.iri]);
+      .spyOn(solidClientFns, "getThingAll")
```

Comment:
You don't need to mock `getThingAll`, but make sure to provide the dataset you need it to process.

To test the filter function, maybe insert a thing that isn't a `foaf:Person` as well.